frontend/src/bundles/studio/components/video-menu/components/menu-content/avatars-content/avatars-content.tsx

```diff
@@ -9,7 +9,7 @@ const AvatarsContent: React.FC = () => {
     }
     const avatars = [];
     for (let index = 0; index < 10; index++) {
-        avatars.push(<AvatarCard preview={avatar} />);
+        avatars.push(<AvatarCard preview={avatar} key={index} />);
```

Comment:
Maybe instead of {avatars} you can use something like this?
```
                {[...Array(10).keys()].map((key) => (
                    <AvatarCard preview={avatar} key={key} />
                ))}
```

Reply:
This is mock data which will be removed in task https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/153